Codecademy/gamut
# feat(Icons): add EmailFilledIcon, WhatsAppIcon, and XIcon

### Overview


Adds EmailedFilledIcon and vendor icons WhatsAppIcon and XIcon


### PR Checklist

- [x] Related to designs: https://www.figma.com/file/mBNCdtMC9HrrTFqCZghPgP/Referral-Rock-Implementation-v1?node-id=233%3A31917&mode=dev

#### Testing Instructions



1. Use the [Storybook preview](https://659dd43f514f1b65c092c3c6--gamut-preview.netlify.app/?path=/docs/atoms-icons-regular--editable-icon) to view the new icons
2. Ensure the icons look good in the preview
- `EmailFilledIcon` will be under "Interface Icons"
- `WhatsAppIcon` and `XIcon` will be under "Vendor Icons"

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/4636)  | [Portal Env](https://tengu.codecademy.com/learn/learn-python-3?PR_ENV=portal-app-pr-4636)   |
